test/e2e/framework/helper/featureset/featureset.go

```diff
@@ -130,4 +130,9 @@ const (
 	// certificates with an Ed25519 private key. This is useful for some
 	// issuers that have may not support this feature.
 	Ed25519FeatureSet Feature = "Ed25519"
+
+	// CheckValidACMEDNSName denotes whether the target is able to
+	// check that the requested certificate DNS Name is a valid
+	// ACME DNS Name.
+	CheckValidACMEDNSName = "CheckValidACMEDNSName"
```

Comment:
Could you expand on this comment to give some examples as to what makes a DNS Name not ACME valid? I wasn't aware of any DNS names which aren't ACME valid.